test/test.js

```diff
@@ -646,6 +641,54 @@ test('assertions return promises', t => {
 	});
 });
 
+// https://github.com/avajs/ava/issues/1330
+test('no crash when adding assertions after the test has ended', t => {
+	t.plan(3);
+
+	ava(a => {
+		a.pass();
+		setTimeout(() => {
```

Comment:
@sindresorhus It's not OK, that's what #1330 is about. This merely verifies that the internal handling doesn't cause a crash, since the error cannot currently be communicated.